### ISSUE TITLE
LORA

### ISSUE THREAD

**Issue by erlebach** [labmlai/annotated_deep_learning_paper_implementations#263]
An implementation of LORA and other tuning techniques would be nice.

**Comment by vpj** [comment]
Yeah that's what we've been planning to add. We are doing some cleanups on the current code base right now. Will start the Lora implementation in a week or two.

**Comment by vpj** [comment]
https://github.com/labmlai/annotated_deep_learning_paper_implementations/pull/266
